src/components/plots/plot.ts

```diff
@@ -54,6 +56,8 @@ module Plottable {
       this.clipPathEnabled = true;
       this.classed("plot", true);
       this._key2PlotDatasetKey = d3.map();
+      this._attrToExtents = d3.map();
+      this._extentProvider = (scale: Scale<any, any>) => this._extentsForScale(scale);
```

Comment:
I could pass `extentsForScale` directly as the `ExtentProvider`, but the `this`-context would be wrong.